src/account.ts

```diff
@@ -1,23 +1,124 @@
-import * as ethjsUtil from 'ethjs-util'
+import * as assert from 'assert'
+import * as BN from 'bn.js'
+import * as rlp from 'rlp'
+import { stripHexPrefix } from 'ethjs-util'
+import { KECCAK256_RLP, KECCAK256_NULL } from './constants'
+import { zeros, bufferToHex, toBuffer, unpadBuffer } from './bytes'
+import { keccak, keccak256, keccakFromString, rlphash } from './hash'
+import { assertIsHexString, assertIsBuffer } from './helpers'
+
 const {
   privateKeyVerify,
   publicKeyCreate,
   publicKeyVerify,
   publicKeyConvert,
 } = require('ethereum-cryptography/secp256k1')
-import * as assert from 'assert'
-import * as BN from 'bn.js'
-import { zeros, bufferToHex } from './bytes'
-import { keccak, keccak256, keccakFromString, rlphash } from './hash'
-import { assertIsHexString, assertIsBuffer } from './helpers'
 
-/**
- * Returns a zero address.
- */
-export const zeroAddress = function(): string {
-  const addressLength = 20
-  const addr = zeros(addressLength)
-  return bufferToHex(addr)
+export interface AccountData {
+  nonce?: BNLike
+  balance?: BNLike
+  stateRoot?: BufferLike
+  codeHash?: BufferLike
+}
+
+type BNLike = BN | string | number
+
+type BufferLike = Buffer | TransformableToBuffer | PrefixedHexString | number
```

Comment:
I think it would be handy for future integrations to also export these types

Reply:
sure good idea, done in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/3f858dbc5e58e7e86dbbe365b70b48244407b2d5